client/src/helpers/format.test.js

```diff
@@ -16,11 +16,6 @@ test('toFormattedCurrency', () => {
 
 describe('toFormattedDate', () => {
 	test('valid date', () => {
-		expect(toFormattedDate('10-10-2021', 'ja-JP')).toEqual('2021/10/10 0:00:00');
-	});
-	test('invalid date', () => {
-		const mockDate = new Date('10-10-2021 01:00');
-		global.Date = jest.fn().mockImplementation(() => mockDate);
-		expect(toFormattedDate('test', 'en-US')).toEqual('10/10/21, 01:00:00');
+		expect(toFormattedDate('10-10-2021 01:00:00', 'en-US')).toEqual('10/10/21, 01:00:00');
```

Comment:
It's ok now. Cheers. 

<img width="400" alt="Screen Shot 2021-12-01 at 11 40 31" src="https://user-images.githubusercontent.com/4486806/144172994-8c6d84ed-8dbc-4c7c-85aa-7be31ec60ce8.png">